plugins/receiver/grpclog/receive_service.go

```diff
@@ -0,0 +1,58 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package grpclog
+
+import (
+	"io"
+	"time"
+
+	common "skywalking/network/common/v3"
+	logging "skywalking/network/logging/v3"
+
+	"github.com/apache/skywalking-satellite/protocol/gen-codes/satellite/protocol"
+)
+
+const eventName = "grpc-log-event"
+
+type LogReportService struct {
```

Comment:
Could you rename the `receive_service.go` according to this? Same request for receiver.go. By the rule you are using now, we will face duplicated names in all plugins.

Reply:
no problem